src/core/components/layout-utils.jsx

```diff
@@ -135,6 +135,9 @@ export class Select extends React.Component {
     className: PropTypes.string,
     disabled: PropTypes.bool,
     description: PropTypes.string,
+
```

Comment:
```suggestion
```
An empty line here I think. Also, can we camelcase these prop names?

Reply:
This empty line is used to separate the "component props" and "DOM native aria props". Since the `aria-*` series are well-known DOM attributes, would be mapped to the native `<select />` element directly and the props in `<select />` are even kebab-case [at React native level](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d304ee67cd3b0abbf52a032f14e885949466b100/types/react/index.d.ts#L1665), I'd like to keep them kebab-case.